g3/generate/pr/prompts/creator.py

```diff
@@ -1,8 +1,59 @@
 from typing import Optional
 
 from g3.domain.message_tone import MessageTone
+from g3.generate.pr.prompts.template import pr_template
+from g3.git.gitinfo import GitInfo
+from g3.main import config
 
 
 class Creator:
-    def create(self, tone: MessageTone, jira: Optional[str] = None, include: Optional[str] = None) -> list:
-        raise NotImplementedError
+    def __init__(self):
+        self.git_info = GitInfo()
+
+    def create(self, tone: MessageTone, include: Optional[str] = None) -> list:
+        system_messages = self.create_system_messages(tone, include)
+        return system_messages + self.user_messages
+
+    @property
+    def user_messages(self) -> list:
+        commits: list = []  # TODO: replace with actual commits for git info
+        content = f"""
+        f"Please provide a pull request description for the provided commit messages. Commits: ```{commits}```."
+         The commit messages are form a git branch named {self.git_info.branch}.
+         The commit messages are from a git repository named {self.git_info.repo}."""
+
+        return [
+            {
+                "role": "user",
+                "content": content.replace("\n", ""),
+            }
+        ]
+
+    def create_system_messages(self, tone: MessageTone, include: Optional[str] = None) -> list:
+        content = "You are a helpful assistant creating Pull Request descriptions for a"
```

Comment:
How about using multi-line strings instead for better visibility like [here](https://github.com/Workable/g3/blob/main/g3/generate/commit/prompts/creator.py#L48-L65)? Mark the one leading whitespace without any trailing one.